staging/src/k8s.io/api/discovery/v1/well_known_labels.go

```diff
@@ -25,4 +25,8 @@ const (
 	// same cluster. It is highly recommended to configure this label for all
 	// EndpointSlices.
 	LabelManagedBy = "endpointslice.kubernetes.io/managed-by"
+	// LabelSkipMirror can be set to true on an Endpoints resource to indicate
+	// that the EndpointSliceMirroring controller should not mirror this
+	// resource with EndpointSlices.
+	LabelSkipMirror = "endpointslice.kubernetes.io/skip-mirror"
```

Comment:
Why is this in here?  Should it be a different commit?

Reply:
Also something that made it to [v1beta1](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/discovery/v1beta1/well_known_labels.go#L31) but not v1alpha1.